src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -0,0 +1,190 @@
+import logging
+from datetime import timedelta
+
+from django.conf import settings
+from django.utils import timezone
+from rest_framework import status
+from urllib3 import BaseHTTPResponse
+from urllib3.exceptions import MaxRetryError, TimeoutError
+
+from sentry import features
+from sentry.conf.server import SEER_ANOMALY_DETECTION_STORE_DATA_URL
+from sentry.incidents.models.alert_rule import AlertRule, AlertRuleThresholdType
+from sentry.models.user import User
+from sentry.net.http import connection_from_url
+from sentry.seer.anomaly_detection.types import ADConfig, Alert, StoreDataRequest, TimeSeriesPoint
+from sentry.seer.signed_seer_api import make_signed_seer_api_request
+from sentry.snuba.models import SnubaQuery
+from sentry.snuba.referrer import Referrer
+from sentry.snuba.utils import get_dataset
+from sentry.utils import json
+from sentry.utils.snuba import SnubaTSResult
+
+logger = logging.getLogger(__name__)
+
+seer_anomaly_detection_connection_pool = connection_from_url(
+    settings.SEER_ANOMALY_DETECTION_URL,
+    timeout=settings.SEER_ANOMALY_DETECTION_TIMEOUT,
+)
+
+
+def format_historical_data(data: SnubaTSResult) -> list[TimeSeriesPoint]:
+    """
+    Format Snuba data into the format the Seer API expects.
+    If there are no results, it's just the timestamp
+    {'time': 1719012000}, {'time': 1719018000}, {'time': 1719024000}
+
+    If there are results, the count is added
+    {'time': 1721300400, 'count': 2}
+    """
+    formatted_data = []
+    for datum in data.data.get("data", []):
+        ts_point = TimeSeriesPoint(timestamp=datum.get("time"), value=datum.get("count", 0))
+        formatted_data.append(ts_point)
+    return formatted_data
+
+
+def translate_direction(direction: int) -> str:
+    """
+    Temporary translation map to Seer's expected values
```

Comment:
temp because we can ask the ai/ml team to update what the expect :) but for now we'll just translate

Reply:
Should I copy this over to the subscription processor? If we expect that this is a permanent fix, should I import it from this file?

Reply:
Maybe we should just ask Ram to update the Seer side to expect ints and import the `AlertRuleThresholdType`